docs/user/ppl/cmd/eventstats.rst

```diff
@@ -221,17 +221,16 @@ Usage: VAR_POP(expr). Returns the population standard variance of expr.
 
 Example::
 
-    PPL> source=accounts | eventstats var_pop(age);
+    os> source=accounts | fields account_number, gender, age | eventstats var_pop(age) | sort account_number;
```

Comment:
Just wonder is there any way to avoid adding a sort in each example query? Why some queries below doesn't require sorting?